src/FRP/Event.js

```diff
@@ -157,19 +157,24 @@ function subject() {
       };
     },
     push: function(a) {
-      for (var i = 0; i < subs.length; i++) {
-        subs[i](a);
-      }
+      return function() {
+        for (var i = 0; i < subs.length; i++) {
+          subs[i](a);
+        }
+      };
     }
   };
 };
 
 exports.fix = function(f) {
-  var s = subject();
+  var s = exports.create();
```

Comment:
This might not work well with dead code elimination. Could you please test it and see?

Reply:
It works with `pulp browserify -m Test.Main --optimise --to html/index.js`, which I think does DCE right?

Reply:
Yes, thanks!